### PR TITLE
Create Explore View

### PR DESCRIPTION
**Pull request by RCAVelez**
**What's new:**
The explore view is created with the attractions panel, the map, and the show route button
On attraction hover the opacity changes

**What's changed:**
The map size

**How was it changed:**
In map.css changed the height and width of the map

Screen Shot 2020-06-29 at 12 24 26 PM

### CODE REVIEW

File: frontend/src/attraction/Attraction.js [hunk 1]
```diff
@@ -7,9 +7,7 @@ import styles from './Attraction.module.css';
  */
 function Attraction({imageUrl}) {
   return (
-    <div className={styles['attractionContainer']}>
-      <img src={imageUrl}/>
-    </div>
+    <img className={styles['attraction']} src={imageUrl}/>
```
Comment (Cindyzhang977) [inline frontend/src/attraction/Attraction.js:10]: Are we using `styles['attraction']` or `styles.attraction`?

Comment (RCAVelez) [reply frontend/src/attraction/Attraction.js:10]: styles.attraction

File: frontend/src/App.js [hunk 2]
```diff
@@ -12,10 +14,7 @@ function App() {
 
   return (
     <div>
-      <Button onClick={handleClick} variant="secondary">
-        Click Me!
-      </Button>
-      <div>{text}</div>
+      <Explore images={[imgTest, imgTest, imgTest]} />
```
Comment (genedwards) [inline frontend/src/App.js:17]: Shouldn't we remove this from the PR?

File: frontend/src/explore/Explore.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import styles from './Explore.module.css';
+import AttractionsPanel from '../attractions-panel/AttractionsPanel';
+import NavButton from '../navigation/NavButton';
+import Map from '../map/Map';
+
+/**
+ * Explore view with attraction selection and map
+ * images is array of urls for an image
+ */
+
```
Comment (genedwards) [inline frontend/src/explore/Explore.js:11]: nit: remove blank line

File: frontend/src/index.js [hunk 4]
```diff
@@ -4,12 +4,7 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+ReactDOM.render(<App />, document.getElementById('root'));
```
Comment (genedwards) [inline frontend/src/index.js:7]: Don't change this unless there's a reason to.

File: frontend/src/explore/Explore.js (hunk 3 above)
Comment (RCAVelez) [reply frontend/src/explore/Explore.js:11]: removed

File: frontend/src/index.js (hunk 4 above)
Comment (RCAVelez) [reply frontend/src/index.js:7]: okidoki

File: frontend/src/explore/Explore.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+function Explore({images}) {
+  return (
+    <div className={styles['exploreContainer']}>
+      <div className={styles['attractionsSection']}>
+        <AttractionsPanel attractionUrls={images}/>
+        <NavButton className={styles['routeButton']} link={'/route'} label={'Show Route'}/>
+      </div>
+      <div className={styles['mapContainer']}>
+        <Map destinations={''}/>
```
[the remark this answers is not in the document]
Comment (RCAVelez) [reply frontend/src/explore/Explore.js:20]: done

File: frontend/src/App.js (hunk 2 above)
Comment (RCAVelez) [reply frontend/src/App.js:17]: removed

**Comment by zoebarth** [comment]
Hey make sure you run prettier before you merge your changes into master.

File: frontend/src/attractions-panel/AttractionsPanel.js [hunk 6]
```diff
@@ -5,14 +5,14 @@ import styles from './AttractionsPanel.module.css';
 /**
  * AttractionsPanel shows all attractions.
  * Updates browser url when an image selection is toggled
- * attractionUrls is an array of urls of type String
+ * @param {list} attractionUrls list of img urls
```
Comment (mecolmg) [inline frontend/src/attractions-panel/AttractionsPanel.js:8]: nit: The type here should be `string[]`

File: frontend/src/attraction/Attraction.js [hunk 7]
```diff
@@ -3,13 +3,11 @@ import styles from './Attraction.module.css';
 
 /**
  * Attraction contains the image of the attraction.
- * imageUrl contains source of image.
+ * @param {String} imageUrl url of image
```
Comment (mecolmg) [inline frontend/src/attraction/Attraction.js:6]: nit: `string` should be lowercase. 
https://jsdoc.app/tags-type.html

File: frontend/src/attractions-panel/AttractionsPanel.js (hunk 6 above)
Comment (RCAVelez) [reply frontend/src/attractions-panel/AttractionsPanel.js:8]: got it

File: frontend/src/attraction/Attraction.js (hunk 7 above)
Comment (RCAVelez) [reply frontend/src/attraction/Attraction.js:6]: done

**Pull request merged**